documents/transfers.md

```diff
@@ -0,0 +1,691 @@
+    "rrn": null
+  },
+  "created_at": 1580454723
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Fetch payments of a linked account
+
+```rb
+Razorpay.headers = {"X-Razorpay-Account" => "linkedAccountId"}
+
+Razorpay::Payment.all
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| X-Razorpay-Account   | string      | The linked account id to fetch the payments received by linked account |
+
+**Response:**
+```json
+{
+  "entity": "collection",
+  "count": 2,
+  "items": [
+    {
+      "id": "pay_E9uth3WhYbh9QV",
+      "entity": "payment",
+      "amount": 100,
+      "currency": "INR",
+      "status": "captured",
+      "order_id": null,
+      "invoice_id": null,
+      "international": null,
+      "method": "transfer",
+      "amount_refunded": 0,
+      "refund_status": null,
+      "captured": true,
+      "description": null,
+      "card_id": null,
+      "bank": null,
+      "wallet": null,
+      "vpa": null,
+      "email": "",
+      "contact": null,
+      "notes": [],
+      "fee": 0,
+      "tax": 0,
+      "error_code": null,
+      "error_description": null,
+      "created_at": 1580219046
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Reverse transfers from all linked accounts
+
+```rb
+transferId = "trf_EAznuJ9cDLnF7Y"
+
+para_attr = {
+    "amount":100
+}
+
+Razorpay::Transfer.fetch(transferId).reverse(para_attr)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| transferId*   | string      | The id of the transfer to be fetched  |
+| amount   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
+
+**Response:**
+```json
+{
+  "id": "rvrsl_EB0BWgGDAu7tOz",
+  "entity": "reversal",
+  "transfer_id": "trf_EAznuJ9cDLnF7Y",
+  "amount": 100,
+  "fee": 0,
+  "tax": 0,
+  "currency": "INR",
+  "notes": [],
+  "initiator_id": "CJoeHMNpi0nC7k",
+  "customer_refund_id": null,
+  "created_at": 1580456007
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Hold settlements for transfers
+```rb
+
+paymentId = "pay_EB1R2s8D4vOAKG" 
+
+para_attr = {
+  "transfers": [
+    {
+      "amount": 100,
+      "account": "acc_I0QRP7PpvaHhpB",
+      "currency": "INR",
+      "on_hold": 1
+    }
+  ]
+}
+
+Razorpay::Payment.fetch(paymentId).transfer(para_attr)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| transferId*   | string      | The id of the transfer to be fetched  |
```

Comment:
paymentId and not transferId

Reply:
@ankitdas13

Reply:
https://github.com/razorpay/razorpay-ruby/blob/update_transfers/documents/transfers.md#hold-settlements-for-transfers